document/js-api/index.bs

```diff
@@ -1113,7 +1115,7 @@ Note: Exported Functions do not have a \[[Construct]] method and thus it is not
                 1. Let |type| be the [=JavaScript exception tag=].
                 1. Let |payload| be « ».
             1. Let |opaqueData| be [=ToWebAssemblyValue=](|v|, [=externref=])
-            1. [=WebAssembly/Throw=] with |type|, |payload| and |opaqueData|.
+            1. [=WebAssembly/Throw=] with |type|, |payload|, associating |opaqueData| with the thrown exception.
```

Comment:
How? Also I'm not sure what exactly is "the thrown exception". Is that a combination of a tag and a payload? Do we have an object that represents that combination which can be used as a key to some map? (And where do we have that map? Don't we only have a map of externaddr -> object?)

Reply:
This "association" is defined a little more down in the definition of Throw below.
(but really it's the hand-wavy bit).
But I just realized I wrote the definition of throw as taking `opaqueData` as an argument, so there's no need to use the term "association" or "thrown" exception here. It should just say
`[=WebAssembly/Throw=] with |type|, |payload|, and |opaqueData|` as it did before, and then the definition down below should say what the association means. I'll make that update here, and put more replies down there.